### PR TITLE
Update Thomas Graves Information

### PR DESCRIPTION
**Pull request by tgravescs**
Update my company info.

I ran jekyll build and server and it looked fine.

### CODE REVIEW

**Comment by tgravescs** [comment]
@srowen @squito have time for a quick review?

**Review by HyukjinKwon — APPROVED** [review]

**Comment by HyukjinKwon** [comment]
I'm pretty sure it can be just pushed .. :D..

File: committers.md [hunk 1]
```diff
@@ -26,7 +26,7 @@ navigation:
 |Robert Evans|Oath|
 |Wenchen Fan|Databricks|
 |Joseph Gonzalez|UC Berkeley|
-|Thomas Graves|Oath|
+|Thomas Graves|Nvidia|
```
Comment (srowen) [inline committers.md]: Super nit but isn't the company name NVIDIA :)

Comment (tgravescs) [reply committers.md]: Yes it should be.. thanks :)

**Review by felixcheung — APPROVED** [review]

**Pull request merged**